basilisk/tests/test_main_app.py

```
@@ -0,0 +1,280 @@
+import pytest
```

Ruff (0.11.9)

Missing docstring in public module

(D100)